mat-process-api.Tests/V1/Helper/ImagePersistingHelperTests.cs

```diff
@@ -0,0 +1,35 @@
+using Bogus;
+using mat_process_api.V1.Helpers;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace mat_process_api.Tests.V1.Helper
+{
+    [TestFixture]
+    public class ImagePersistingHelperTests
+    {
+        Faker faker;
+        ImagePersistingHelper helper;
+        [SetUp]
+        public void set_up()
+        {
+            faker = new Faker();
+        }
+        [TestCase("2021/01/abc-123-bbb")]
```

Comment:
should we update the test cases to follow the new format i.e. just the image id (abs-123-bbb) without the year/month/ part?